netbout-web/src/main/java/com/netbout/rest/bout/TkIndex.java

```diff
@@ -79,10 +80,12 @@ public Response act(final Request req) throws IOException {
         );
         final Bout bout = new RqBout(this.base, req).bout();
         final Href home = new Href("/b").path(bout.number());
+        final RqWithDefaultHeader reqdef =
```

Comment:
@exper0 the reason here is the compound name rule http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

We use the scope to own advantage. Since the incoming request is `req`, the name `request` for our temporary variable is free in this scope.